daemon/algod/api/server/v2/handlers.go

```diff
@@ -732,6 +732,54 @@ func (v2 *Handlers) GetBlockTxids(ctx echo.Context, round uint64) error {
 	return ctx.JSON(http.StatusOK, response)
 }
 
+func getLogsFromTxns(txns []transactions.SignedTxnWithAD, blockLogs *[]model.BlockLog, outerTxnID string) {
+
+	for _, txn := range txns {
+		byteLogs := [][]byte{}
+		for _, log := range txn.EvalDelta.Logs {
+			byteLogs = append(byteLogs, []byte(log))
```

Comment:
And then you would:
```suggestion
		for i, log := range txn.EvalDelta.Logs {
			byteLogs[i] = []byte(log)
```

Reply:
More importantly, you're going to have to go digging through that `EvalDelta` to find all the logs, if you want them for inners of inners too.